XenModel/Messages.zh-CN.resx

```diff
@@ -3623,19 +3623,19 @@
     <value>yyyy-MM-dd</value>
   </data>
   <data name="DATEFORMAT_DMY_HM" xml:space="preserve">
-    <value>yyyy-MM-dd tt h:mm</value>
+    <value>yyyy-MM-dd HH:mm</value>

```

Comment:
I don't understand why this has been changed to the 24-hour format, while DATEFORMAT_HM has been changed to 12-hour format on line 3635. 

This change wasn't needed for CA-236057 and will create discrepancies again on the Performance tab and on other windows where both formats are used.